src/status_im/browser/core.cljs

```diff
@@ -420,7 +420,10 @@
  :browser/send-to-bridge
  (fn [{:keys [message webview]}]
    (when (and message webview)
-     (.sendToBridge webview (types/clj->json message)))))
+     (let [msg (str "ReactNativeWebView.onMessage('"
```

Comment:
could you elaborate ?  looks weird

Reply:
It's a community-webview way of posting messages in RN->WebView direction. I used a modified version of what was suggested [here](https://github.com/react-native-community/react-native-webview/issues/809#issuecomment-527245591).
We invoke `onMessage` fn that is defined in `provider.js`